multimodal/src/autogluon/multimodal/models/ft_transformer.py

```diff
@@ -396,6 +509,8 @@ def __init__(
         head_normalization: ModuleType,
         d_out: int,
         projection: Optional[bool] = False,
+        additive_attention: Optional[bool] = False,
+        share_qv_weights: Optional[bool] = False,
```

Comment:
Need docstrings for additive_attention and share_qv_weights.

Reply:
It would be good if you could add all the missing docstrings. Yiqing forgot doing so in his PR.

Reply:
This is added. Please check